src/model/inners.js

```diff
@@ -1,7 +1,5 @@
 import { Node } from './node'
 
-export const Assignment = (variable, value) => Node('Assignment')({ variable, value })
-
-export const Catch = (variable, type = null) => (...handler) => Node('Catch')({ variable, type, handler })
+export const Catch = (variable, type) => (...handler) => Node('Catch')({ variable, type, handler: Block(...handler) })
 export const Parameter = (name, varArg = false) => Node('Param')({ name, varArg })
-export const SuperType = (name) => (...parameters) => Node('SuperType')({ name, parameters })
\ No newline at end of file
+export const Block = (...sentences) => Node('Block')({ sentences })
```

Comment:
Cool, I will use this for the linker. I think after a refactor that I'm doing to support classes references I'll want to also merge since I'm diverging too much, and I have something already "working"